packages/network/lib/agent.ts

```diff
@@ -467,6 +467,15 @@ class HttpsAgent extends https.Agent {
   }
 }
 
+// This agent honors NODE_TLS_REJECT_UNAUTHORIZED (though this is overridden
```

Comment:
Do we want to call out that it is currently set to '0' and to not use this agent for any interactions with the Cloud?

Reply:
yeah good call

Reply:
[`73913d9` (#32624)](https://github.com/cypress-io/cypress/pull/32624/commits/73913d938e29a5aab57a9ccea2f8d89f08be7799)